lib/stylesheets/bootstrap.js

```diff
@@ -4,6 +4,7 @@
 
 */
 'use strict';
+import {Platform} from 'react-native';
```

Comment:
Please leave an empty line between `use strict` and the `import. And also add spaces between `Platform`. Just to keep in line with the rest of the code style.